### PR TITLE
[Kan-25] Retrieve API secrets from env file

### PR DESCRIPTION
**Pull request by MinhTeeFresh**

### CODE REVIEW

**Comment by MinhTeeFresh** [comment]
I may need to add some comments to the code, but I primarily wanted a review on the README changes, does it make sense for new users?

File: README.md [hunk 1]
```diff
@@ -1,11 +1,25 @@
...
 pip freeze > requirements.txt
 pip install requirements.txt
-```
\ No newline at end of file
+```
+
+## Creating .env file
+
+For purposes of having using FAA API, users must utilize their own API keys in a seperate .env file </br>
+This can be done by creating a .env file in the root directory </br>
```
Comment (tanminivan) [inline README.md:19]: Did you add the `.env` file to the `.gitignore`?

**Review by tanminivan — CHANGES_REQUESTED** [review]

**Review by tanminivan — COMMENTED** [review]
Did this PR automatically add reviewers or did you have to review them manually, @MinhTeeFresh ? (great username btw)

**Comment by tanminivan** [comment]
Man, I hit the wrong thing, I just wanted to add a comment, but requested changes instead, now it appears I can't change my vote? Thanks github.

**Review by JamesJWilliams3000 — APPROVED** [review]

**Comment by MinhTeeFresh** [comment]
> Did this PR automatically add reviewers or did you have to review them manually, @MinhTeeFresh ? (great username btw)

No, I manually added the reviewers

**Review by aidanperkins — APPROVED** [review]

**Review by Zombiepro89 — APPROVED** [review]

**Review by Zenpock — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (tanminivan) [reply README.md:19]: Ah, yeah, you are correct. It's already being ignored. Don't forget to squash and rebase.

**Review by tanminivan — APPROVED** [review]

**Pull request merged**